### DISCUSSION TITLE
AI chatbot in Personal Messaging

### DISCUSSION THREAD

**Discussion by PrajjwalDatir** [PrajjwalDatir/Kaoi#23] [Ideas]
If you have any command requests or ideas.
Something which you wanna share!
Drop it here.

**Comment by ghost** [comment]
How about a talking bot with ai intelligence?
Is it possible?

**Reply by PrajjwalDatir** [reply]
**Yes it's possible.**

I'm looking forward to do it in few months.

**Reply by PrajjwalDatir** [reply]
> How about a talking bot with ai intelligence?
> Is it possible?

@Sahib111  It's implemented now!

**Comment by PrajjwalDatir** [comment]
Anyone having coding experience, can try integrating [this](https://rapidapi.com/hub) APIs in the Bot

**Comment by PrajjwalDatir** [comment]
Done
